build_tools/benchmarks/comparisons/setup_desktop.sh

```diff
@@ -0,0 +1,110 @@
+#!/bin/bash
+
+# Install Bazel. From https://www.tensorflow.org/install/source
+npm install -g @bazel/bazelisk
```

Comment:
:fearful: node?!

Reply:
I hate node, and it seems especially weird for a non-JS thing. I guess it's like python devs will pip install everything. It looks like that's the only single-command package-manager option for Bazelisk on Linux. It does add an unnecessary npm dependency here though, and, for instance, I don't have npm installed on either my laptop or my workstation. Anyway, if you think this is the best way to install it, then that's fine. We also don't use Bazelisk in IREE usually. Not that it's bad. But it might be simpler to just install Bazel here. IDK